noir-projects/noir-protocol-circuits/crates/types/src/abis/public_inner_call_request.nr

```diff
@@ -0,0 +1,60 @@
+use crate::{
+    abis::{public_call_stack_item_compressed::PublicCallStackItemCompressed, side_effect::Ordered},
+    constants::PUBLIC_INNER_CALL_REQUEST_LENGTH, traits::{Empty, Serialize, Deserialize},
+    utils::reader::Reader
+};
+
+// TODO(#7124): To be deprecated.
+struct PublicInnerCallRequest {
+    item: PublicCallStackItemCompressed,
+    counter: u32,
+}
```

Comment:
Will the public circuit public inputs also be deprecated/removed when the AVM is truly processing an entire enqueued call?

Reply:
Yes it can also be removed at the time 😃